com.sap.cloud.lm.sl.cf.web/src/test/java/com/sap/cloud/lm/sl/cf/web/api/impl/MtaApiServiceImplTest.java

```diff
@@ -32,27 +34,27 @@
 
     @Mock
     private CloudControllerClientProvider clientProvider;
-
     @Mock
-    private AuthorizationChecker authorizationChecker;
-
+    private CloudControllerClient client;
     @Mock
-    private HttpServletRequest request;
-
+    private Authentication auth;
     @Mock
-    private CloudControllerClient client;
+    private SecurityContext securityContext;
+    @Spy
+    private DeployedComponentsDetector componentsDetector;
 
     @InjectMocks
     private MtasApiServiceImpl testedClass;
 
-    List<CloudApplication> apps;
-    List<Mta> mtas;
+    private List<CloudApplication> apps;
+    private List<Mta> mtas;
 
     private static final String USER_NAME = "someUser";
-    private static final String SPACE_GUID = "896e6be9-8217-4a1c-b938-09b30966157a";
+    private static final String SPACE_GUID = UUID.randomUUID()
+                                                 .toString();
 
-    @Before
-    public void initialize() throws Exception {
+    @BeforeEach
```

Comment:
is the @BeforeEach change necessary?

Reply:
This is just migrating from JUnit 4 to 5. It's not a change in behaviour.

Reply:
oh, yes, you're right